Cargo.toml

```diff
@@ -11,10 +11,13 @@ version = "0.18.0"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
+futures = "0.1.16"
```

Comment:
This should only be used when `reqwest` feature is enabled. Do you know how to do this @dten ?

Reply:
in the toml you make make the dependency optional and define a feature
`unstable = ["futures"]`
then your unstable features should be hidden behind "request" _and_ "unstable"